src/sagas/index.js

```diff
@@ -134,13 +134,16 @@ function* selectSideEffects({ payload }) {
 }
 
 function* saveProjectMetadata({ metadata }) {
-  console.log(metadata);
   const { manifestLabel, manifestSummary } = metadata;
   yield put(setTitle(manifestLabel));
   yield put(setDescription(manifestSummary));
   yield put(cancelProjectMetadataEdits());
 }
 
+function* afterDelete() {
+  yield put(deleteRedundantSizes());
```

Comment:
@adam-digirati Just quickly, I'm not asking to change how you're doing this as its fine. Sagas are a great place to write logic, such as finding which bubbles to delete. Redux-saga can reduce complexity in your reducers by extending the functionality of your application, without changing the basic functions of your reducer. When A to X, Y and Z.